app/presenters/search_payload_presenter.rb

```diff
@@ -0,0 +1,36 @@
+class SearchPayloadPresenter
+  attr_reader :registerable_edition
+  delegate :slug,
+           :title,
+           :description,
+           :indexable_content,
+           :public_timestamp,
+           :sections,
+           :artefact,
+           :format,
+           to: :registerable_edition
+
+  def initialize(registerable_edition)
+    @registerable_edition = registerable_edition
+  end
+
+  def self.present(registerable_edition)
+    new(registerable_edition).present
+  end
+
+  def present
+    {
+      content_id: artefact.content_id,
+      rendering_app: "publisher",
+      publishing_app: "publisher",
+      format: format.underscore,
+      title: title,
+      description: description,
+      indexable_content: indexable_content,
+      link: "/#{slug}",
+      public_timestamp: public_timestamp,
+      specialist_sectors: registerable_edition.specialist_sectors.compact,
+      sections: sections,
```

Comment:
As defined by https://github.com/alphagov/publisher/blob/master/lib/registerable_edition.rb#L40, and picked up as an optional param here: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/panopticon/registerer.rb#L38.

This basically doesn't need to be sent, for the same reason as the specialist sectors - rummager will fetch the links for a published doc.